index.rst

```diff
@@ -372,30 +372,41 @@ c.l.o stubb
 
   Here is where we currently are in the release process. Current step in bold.
 
-  Precursor Steps
+  Release Precursor Steps
   ---------------------------------
 
   1. Identify any pre-release blockers ("must-have features") :tools:
-  2. Wait for them to clear
+  Contributors check if there are outstanding issues that have to be included in the next M.m.p release and related them as blocker to the above issue DM-XXXXX.
+  1. Create Jira issues for each release activity.
```

Comment:
If this is the "above issue", this needs to be before the previous step.  AFAIK there is only one Jira issue, not more than one, so "Create Jira issue for the release activity."

Reply:
This is not the above issue. These issues are different activities that concur to the release, like for example create a release note, validate the weekly, create the characterization report.
In my first proposal, I was quite detailed but, following John's comment, I simplified it.

Reply:
Then "above issue" in line 379 has no apparent referent; instead it looks like the "below issue" in line 384.